clang/test/Modules/Inputs/builtin-headers/system-modules.modulemap

```diff
@@ -49,6 +49,11 @@ module cstd [system] [no_undeclared_includes] {
     export *
   }
 
+  module stdcountof {
+    header "stdcountof.h"
+    export *
+  }
+
```

Comment:
@ian-twilightcoder should this change `clang/test/Modules/builtin-headers.mm` to account for the new header?

Reply:
Yeah we should import the new module with the other builtin ones in that test.